modules/statics/src/coins.ts

```diff
@@ -499,6 +501,9 @@ export const coins = CoinMap.fromCoins([
     '',
     'WXT'
   ),
+  // TODO Set correct contract addresses for WBTC and WETH on Tron main network
+  tronToken('trx:wbtc', 'Tron Wrapped Bitcoin', 8, 'TPamvhorQcFpvtMtYb6qDDJNMsUM1doPty', UnderlyingAsset.WBTC),
```

Comment:
I'm not sure these UnderlyingASsets are right, I'll have to get back to this

Reply:
Ok. I set two example addresses for WBTC and WETH in mainnet.